src/schemathesis/specs/openapi/schemas.py

```diff
@@ -974,29 +943,37 @@ def collect_parameters(
                 )
         return collected
 
-    def get_response_schema(self, definition: dict[str, Any], scope: str) -> tuple[list[str], dict[str, Any] | None]:
-        scopes, definition = resolve_in_scope(self.resolver, definition, scope)
-        options = iter(definition.get("content", {}).values())
-        option = next(options, None)
-        # "schema" is an optional key in the `MediaType` object
-        if option and "schema" in option:
-            # Extra conversion to JSON Schema is needed here if there was one $ref in the input
-            # because it is not converted
-            return scopes, to_json_schema_recursive(
-                option["schema"], self.nullable_name, is_response_schema=True, update_quantifiers=False
-            )
-        return scopes, None
+    def _parse_responses(self, definition: dict[str, Any], scope: str) -> Responses:
+        responses = {}
+        for key, response in definition.get("responses", {}).items():
+            scopes, response = resolve_in_scope(self.resolver, response, scope)
+            options = iter(response.get("content", {}).values())
```

Comment:
There could be more media types ... is it an oversight that the first one is taken? if so, note it and fix separately